.github/workflows/build.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       # step 2
       - name: "Set up QEMU"
         id: qemu
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v3
```

Comment:
The build_arm64-binary action only works on push events, so there is no way to confirm that this issue has been resolved until this PR merge. Therefore, I propose to temporarily add build_arm64-binary action to pull request in this PR and remove it after confirming that there is no problem.

Reply:
https://github.com/leizhiyuan/seata/actions/runs/6582808710/job/17884865374 
I will test in my fork

Reply:
https://github.com/leizhiyuan/seata/actions/runs/6582999974 (to speed up)

the waring has been resolved, To accelerate the verification process, I mock some command scripts that are unrelated to warnings.

Reply:
ok, it seems to be working fine.